### PR TITLE
Consuming API and displaying Items

### PR DESCRIPTION
**Pull request by JbirdL86**
# What we have Accomplished 🏆 👍 : 
* Consume API from TVMAZE public API ✔️ 
* Consume API from notion API ✔️ 
* Use ES6 Syntax ✔️ 
* Use ES6 Modularity ✔️ 
* Display Elements in the Screen and Also the Counter of the elements ✔️ 
* Add likes to every show and save it in the API ✔️ 
* Add comments and save in the remote API and display the comment's counter ✔️ 
* Implement Jest Testing ✔️ 

## Built with 🔨 : 
- Javascript
- Webpack
- HTML
## Contributors 🤝 : 
@elassis 
@JbirdL86

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 [Watch Video](https://drive.google.com/file/d/1-ugp6XQNO4kUDtyqxslmk6QjYmpHex-x/view?usp=sharing)
 
 
+
 ## Prerrequisites ☑️
```
Comment (dasileker) [inline README.md]: - [ ] Please add how we can download the repo and run it localoy

File: src/movies.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  showItemsCounter: (arr) => {
+    document.querySelector('#shows-counter').innerHTML = arr.length;
+    return arr.length;
+  },
+  getShows: (objArr) => {
+    const arr = Array.from(objArr);
+    arr.forEach((item) => {
+      fetch(`https://api.tvmaze.com/shows/${item.idTv}`)
+        .then((response) => response.json())
+        .then((json) => renderElement(json, item.idInv));
```
Comment (dasileker) [inline src/movies.js]: - [ ] Your method is good but instead using just the fetch you need to make it `async and await and fetch` method, I did see  you are using down but also should be applied here.

hhkkh

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,315 @@
...
+  width: 100%;
+  height: 20%;
+  display: flex;
+  align-items: center;
+}
+
+.comment-btn {
+  display: block;
+  margin: 8px auto;
+  color: grey;
```
Comment (dasileker) [inline src/style.css]: - [ ] The color is a little lighter, you can make more vibrant,

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,315 @@
...
+  padding: 10px 15px;
+  border-radius: 30px;
+  border: none;
+  background-color: rgb(221, 213, 213);
+  transition: all 300ms ease;
+}
+
+.comment-btn:hover {
+  transform: scale(1.1);
```
Comment (dasileker) [inline src/style.css:157]: - [ ] Add also a bg color when we hoover on it.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @JbirdL86** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### **To Highlight**
- The PR is really impressive.
- The commit are good.
- The linters are passing.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
Hi @elassis & @JbirdL86, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**